### PR TITLE
rakudo: update to 2018.10

### PR DESCRIPTION
**Pull request by atweiden**
- specify exact version number for perl6 dependencies

### CODE REVIEW

File: srcpkgs/nqp/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+version=2018.10
 revision=1
 build_style=configure
 make_check_target=test
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --backends=moar"
 hostmakedepends="perl"
-makedepends="MoarVM"
-depends="MoarVM"
+makedepends="MoarVM=2018.10"
```
Comment (maxice8) [inline srcpkgs/nqp/template]: Dependencies on makedepends and hostmakedepends are not subject to version constraints, they always use the latest version in srcpkgs/

File: srcpkgs/nqp/template [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
+depends="MoarVM=2018.10"
```
Comment (maxice8) [inline srcpkgs/nqp/template]: correct specifier for exact version is `-`

other available are `>=` `<=` `>` `<`

Comment (atweiden) [reply srcpkgs/nqp/template]: `-` was what I went with originally: https://travis-ci.org/void-linux/void-packages/jobs/447580109

I assume I needed to add `_revision`, then

File: srcpkgs/nqp/template [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+version=2018.10
 revision=1
 build_style=configure
 make_check_target=test
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --backends=moar"
 hostmakedepends="perl"
 makedepends="MoarVM"
-depends="MoarVM"
+depends="MoarVM-2018.10_1"
```
Comment (maxice8) [inline srcpkgs/nqp/template]: you can use ${version}_${revision}

**Pull request closed without merging**